src/regfox/responses/regfox_api_get_registration_info_parser.js

```diff
@@ -0,0 +1,329 @@
+
+/**
+ * Brings some of the "dynamic" data objects and places them directly on the main response.
+ * 
+ * So instead of 
+ * {
+ *  id: 1234
+ *  data: [{
+ *    path: "dateOfBirth"
+ *    value: "2022-02-02"
+ *  }]
+ * }
+ * 
+ * We get
+ * {
+ *  id: 1234,
+ *  dataOfBirth: "2022-02-02"
+ * }
+ * 
+ * @param {*} rawResponse see below
+ * @returns 
+ */
+const parseRegfoxGetRegistrationResponse = (rawResponse) => {
+  const dataObject = rawResponse?.data?.reduce((acc, i) => {
+    if (!i.value) {
+      return acc;
+    }
+    return { [i.path]: i.value, ...acc };
```

Comment:
This took me a minute, clever!

Reply:
You should have seen the code I originally found on the internet.

```
return acc[i.path]: i.value, acc;
```
That one took me a very long time to realize whats going on, and rewrote it very quick once I did haha.